cryptotracker/static/CCGraph.js

```diff
@@ -25,23 +25,23 @@ class ChartBuilder {
   }
 
   percentChange(num1, num2) {
-    return (num1 - num2) / num2 * 100
+    return ((num1 - num2) / num2 * 100).toFixed(2)
   }
 
   dollarChange(num1, num2) {
     var x = num1 - num2;
     if (x < 0) {
-      return x;
+      return x.toFixed(2);
     }
     return "+" + String(x.toFixed(2));

```

Comment:
`toFixed` already converts the input to a string so no need to recast into`String()`